codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameDecoder.java

```diff
@@ -421,7 +431,33 @@ private static State getNextState(int type, int length) {
         }
     }
 
-    private static boolean isValidFrameHeader(int streamId, int type, byte flags, int length) {
+    /**
+     * Decode the unknown frame, the default implement just skip the bytes.
+     * Returns ture if parsed something, otherwise false.
+     */
+    @SuppressWarnings("unused")
+    protected boolean decodeUnknownFrame(int frameType, byte flags, ByteBuf buffer) {
+        int numBytes = Math.min(buffer.readableBytes(), length);
+        buffer.skipBytes(numBytes);
+        length -= numBytes;
+        if (length == 0) {
+            state = State.READ_COMMON_HEADER;
+            return true;
+        }
+        return false;
+    }
+
+    @SuppressWarnings("unused")
+    protected boolean shouldReadUnknownFrame(int type, int length) {
+        return false;
+    }
+
+    @SuppressWarnings("unused")
```

Comment:
This method is used... Seems like the annotation should are removed

Reply:
I was to suppress the `parameter` unused warning, but yes.